common/utils/src/main/resources/error/error-classes.json

```diff
@@ -674,6 +674,12 @@
     ],
     "sqlState" : "23505"
   },
+  "DUPLICATE_ROUTINE_PARAMETER_ASSIGNMENT" : {
+    "message" : [
+      "Call to function <functionName> is invalid because it includes multiple argument assignments to the same name <parameterName>."
+    ],
+    "sqlState" : "42704K"
```

Comment:
I feel that some of my previous comments are naturally collapsed so you can't see them without expanding. I'll copy those to this new PR.

The error code should be 5 characters according to https://github.com/apache/spark/tree/master/common/utils/src/main/resources/error.